vignettes/bootstrap-themes-in-teal.Rmd

```diff
@@ -141,7 +141,7 @@ As expected, our corners are no longer rounded. If we look at our R console, we
 
 #### Apply the customized theme
 
-Once our customization is complete, we will apply the changes to our application. To do this, we use the option `teal.bs_theme` like before but this time we will expand on our `bslib::bs_theme` call to include our changes. Luckily, the arguments that were printed to the R console when running our app in the themer can be plugged right in.
+Once our customization is complete, we will apply the changes to our application. To do this, we use the option `teal.bs_theme` like before but this time we will expand on our `bslib::bs_theme` call to include our changes. Luckily, the arguments that were printed to the R console when running our app in the `themer` can be plugged right in.
```

Comment:
"Themer" is not neither a function, nor a class, nor any variable names. You can't just backtick any word that is not found in the dictionary just because you want to trim the wordlist.

Reply:
keyword 'themer' has been used with backticks in two other places within same the documentation. I added the backticks to keep it consistent and not solely for the purpos of trimming the wordlist.  `themer` here in context  running app inside `run_with_themer()` or `bs_theme_preview()` so I thought its intentional.

Reply:
done [53bec14](https://github.com/insightsengineering/teal/pull/1021/commits/53bec1456cbfd63c79af48209f072e8339ca588f)